static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -281,6 +286,18 @@ define([
                 callbacks[size] && callbacks[size](event, $slot);
             }
         },
+        allAdsRendered = function (slot) {
+            forEach(slots, function (value, key) {
+                if (key === slot) {
+                    slots[key].rendered = true;
+                    return;
+                }
+            });
```

Comment:
but I wouldn't add a new `rendered` property directly to the slot, as we don't own it. Change slots to something like


```
slots = {
    'dfp-ad--top-above-nav': {
        isRendered: true,
        slot: googletag.slot
    },
   ...
}
```

here's docs for `googletag.slot` btw - https://developers.google.com/doubleclick-gpt/reference#googletag.Slot